backend/spec/controllers/enrollments/change_state_spec.rb

```diff
@@ -443,6 +443,56 @@
     end
   end
 
+  describe "validate event for hubee bridge" do
+    include ActiveJob::TestHelper
+
+    let(:enrollment_api_target) { :hubee_portail }
+    let(:enrollment_status) { :submitted }
+    let!(:hubee_portail_subscriber) { create(:user, roles: ["hubee_portail:subscriber"]) }
+
+    before do
+      login(hubee_portail_subscriber)
+    end
+
+    before do
+      allow_any_instance_of(HubeePortailBridge).to receive(:call)
+    end
+
+    after do
+      clear_enqueued_jobs
+      clear_performed_jobs
+    end
+
+    context "when instructeur validate enrollment" do
+      let(:enrollment_status) { :validated }
+      let(:event) { create(:event, name: "validate", enrollment: enrollment, comment: comment, created_at: DateTime.new(2023, 1, 29), updated_at: DateTime.new(2023, 1, 29)) }
+      let(:comment) { "I like trains" }
+
+      let(:enrollment_mailer) do
+        EnrollmentMailer.with(
+          to: enrollment.responsable_metier_email,
+          enrollment_id: enrollment.id,
+          target_api: enrollment.target_api,
+          template_name: "validate"
+        ).notification_email.deliver_later
+      end
+
+      it "#calls HubeePortailBridge" do
```

Comment:
Ici tu ne testes pas du tout le comportement du bridge: en effet tu stub avec `allow_any_instance_of(HubeePortailBridge).to receive(:call)` ce qui fait que tu ne passes jamais dans cette méthode call.

Si tu veux tester effectivement ton bridge, le plus simple est de faire un test du bridge directement, i.e. un `spec/bridges/hubee_portail_bridge_spec.rb` où dedans tu mockes l'appel au token manager, et de là tu vérifies que l'email est bien parti.

D'ailleurs ici le `enrollment_mailer` envoi un email, et tu testes que l'email est bien parti (donc à aucun moment tu ne testes le comportement du bridge, ton code ici pourrait être écrit n'importe où).

Reply:
Bon en fait je retourne a la case zero, le problème c'est qu'on ne sait pas du ce que doit nous retourner le bridge hubee vu que nous n'avons pas d'environnement de test sur lequel travailler pour ce bridge. 
Même si je mock l'appel, je ne suis pas sure de le mocker comme il faut. 

La solution a laquelle j'ai pensé est d'installer la gem VCR mais j'avoue que uniquement pour ce bridge, ça me paraissait overkill. 

c'est pas grave, je vais reprendre le ticket de zero :)